examples/motoko_examples/simple-to-do/canisters/rust/src/lib.rs

```diff
@@ -0,0 +1,130 @@
+use std::cell::RefCell;
+use std::collections::BTreeMap;
```

Comment:
This should be a HashMap to match more closely the Motoko HashMap and the Azle object (which should be rewritten)